discovery_engine_core/bindings/src/lib.rs

```diff
@@ -29,18 +29,22 @@
 pub mod async_bindings;
 pub mod types;
 
-#[async_bindgen::api]
+#[async_bindgen::api(
+    use xayn_discovery_engine_core::document::HistoricDocument;
+)]
```

Comment:
that's nice, i didn't know this was possible 👍  i'll open a pr to shorten the other full paths used in this module.